README.md

```diff
@@ -10,6 +10,7 @@
 - Style them using CSS
 - Apply drag and drop to sort the list
 - To change the status of a task check and uncheck it
+- Implement the CRUD (create, update, delete) methods. All the elements of the user interface are fully functional
 - Understand how to use webpack
 
 ## Built With
```

Comment:
## Getting Started:

Kindly complete the steps to run your app locally.

I don't think once a user git clone your repo, the app will start to run.

Reply:
- [ ] Kindly add all the above steps in your README file